src/main/java/core/basesyntax/bookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -3,11 +3,13 @@
 import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
+import java.util.List;
 
 public record CreateBookRequestDto(@NotBlank String title,
                                    @NotBlank String author,
                                    @NotBlank String isbn,
                                    @NotNull BigDecimal price,
                                    String description,
-                                   String coverImage) {
+                                   String coverImage,
+                                   List<Long> categoryIds) {
```

Comment:
```suggestion
                                   @NotEmpty List<Long> categoryIds) {
```

Reply:
![image](https://github.com/user-attachments/assets/1bb1e124-15fe-479c-b7b8-04c2de78af92)
looking at this hint from the next task, I assume that it's not mandatory for book to have category